### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by Jmuccigr** [libasoles/graph-view-text-size#1]
I'm just getting an error when I try to activate the plugin in the settings. "Failed to load plugin".

Let me know what info I can provide.

MacOS 13.6.9
Obsidian 1.6.7

**Comment by libasoles** [comment]
Hi. Thanks for trying it out and reporting this. 

I guess you downloaded a zip or tar. I've just tried that and indeed fails. The reason is that the zip contain the source code, not a built version 🤔 

That´s strange because I used the official script to build the release:
https://docs.obsidian.md/Plugins/Releasing/Release+your+plugin+with+GitHub+Actions

I'll check on that next week. In the meantime, see that there are three unzipped files here. They are the built version. Only them should be present in the zip/tar. So, if you download and put them in the plugins folder that will work (main.js, styles.css, manifest)
https://github.com/libasoles/graph-view-text-size/releases/tag/1.0.1

<img width="1211" alt="image" src="https://github.com/user-attachments/assets/22689f84-a16d-4eb8-bcd5-3f1d598cec93">

Sorry the packed files are useless. This is my first time releasing packed stuff.
